src/styles/styles.js

```diff
@@ -2841,6 +2840,7 @@ const styles = {
         height: variables.sliderBarHeight,
         borderRadius: variables.sliderBarHeight / 2,
         alignSelf: 'stretch',
+        justifyContent: 'center',
     },
 
     imageCropRotateButton: {
```

Comment:
I think this should be removed as it is no longer required.

Also should be removed from here

https://github.com/Expensify/App/blob/6ee5e7a5afa5c7066368be9cee1f6b80f4699389/src/components/AvatarCropModal/AvatarCropModal.js#L100

Reply:
i have removed  imageCropRotateButton.height from relativeHeight @Santhosh-Sellavel, please check